python/lsst/pipe/tasks/insertFakes.py

```diff
@@ -681,8 +680,7 @@ def cleanCat(self, fakeCat, starCheckVal):
         will be added.
         """
 
-        rowsToKeep = (((fakeCat[self.config.bulgeHLR] != 0.0) & (fakeCat[self.config.diskHLR] != 0.0))
-                      | (fakeCat[self.config.sourceType] == starCheckVal))
+        rowsToKeep = ((fakeCat[self.config.bulgeHLR] != 0.0) & (fakeCat[self.config.diskHLR] != 0.0))
```

Comment:
You've lost a line here from the check.

Reply:
Huh, don't know when that happened. Put it back.